mediathread/assetmgr/views.py

```diff
@@ -448,6 +450,245 @@ def final_cut_pro_xml(request, asset_id):
+
+        user_node['http://xmlns.com/foaf/0.1/name'] = [{"value": data.get(
+            'annotations')[n]['author']['public_name'], "type": "literal"}]
+        user_node['http://www.w3.org/1999/02/22-rdf-syntax-ns#type'] = [
+            {"value": "http://xmlns.com/foaf/0.1/Person", "type": "uri"}]
+        user_node['http://xmlns.com/foaf/0.1/mbox_sha1sum'] = [
+            {"value": hash_or_username, "type": "literal"}]
+        self.export[self.root + '/user/' + hash_or_username] = user_node
+
+        annotation_node = {}
+        annotation_node['http://purl.org/dc/terms/title'] = [
+            {"value": data.get('annotations')[n]['title'],
+             "type": "literal"}]
+        annotation_node['http://purl.org/dc/terms/description'] = [
+            {"value": "This is an annotation", "type": "literal"}]
+        annotation_node['http://rdfs.org/sioc/ns#content'] = [
+            {"value": data.get('annotations')[n]['metadata']['body'],
+             "type": "literal"}]
+        annotation_node[
+            'http://www.w3.org/ns/prov#wasAttributedTo'
+        ] = [{"value": self.root + data.get(
+            'annotations'
+            )[n]['author']['resource_uri'].rstrip('/') +
+                '/user/' + hash_or_username,
+                "type": "uri"}]
+        a_annotation_node = {}
+        a_annotation_node['http://www.openannotation.org/ns/hasBody'] = [
+            {"value": self.root +
+             data.get('annotations')[n]['url'].rstrip('/'),
+             "type": "uri"}]
+        time = ''
+        time += self.root
+        time += data.get('local_url').rstrip('/')
+        time += '#t=npt:'
+        time += str(data.get('annotations')[n]['annotation']['start'])
+        time += ','
+        time += str(data.get('annotations')[n]['annotation']['end'])
+        # try:
+        a_annotation_node[
+                'http://www.openannotation.org/ns/hasTarget'
+            ] = [{"value": time, "type": "uri"}]
+        # except Exception:
+        #     pass
+        a_annotation_node[
+            'http://www.w3.org/1999/02/22-rdf-syntax-ns#type'
+        ] = [
+            {"value": "http://www.openannotation.org/ns/Annotation",
+             "type": "uri"}]
+        anno_urn = 'urn:mediathread:anno' + str(self.anno_num + 1)
+        self.export[anno_urn.rstrip('/')] = a_annotation_node
+        self.anno_num += 1
+
+        tag = []
+        tag = (data.get('annotations')[n]['metadata']['tags'])
+        for k in range(0, len(tag)):
+            self.parse_tag(k, tag, n)
+
+        vocab = []
+        vocab = data.get('annotations')[n]['vocabulary']
+        self.parse_vocab(vocab, n, data)
+
+        try:
+            self.export[self.root + data.get('annotations')[n]
+                        ['url'].rstrip('/')] = annotation_node
+        except Exception:
+            pass
+
+    def parse_vocab(self, vocab, n, data):
+        print vocab
+        for j in range(0, len(vocab)):
+                num = 0
```

Comment:
That is one line that is indented to conform to pep8. I can create a variable to shorten the line.